pages/account_and_service_management/reversibility/23-mutualized-virtualization-vcd-reversibility/guide.en-gb.md

```diff
@@ -21,23 +21,22 @@ Features of the product line fall into three categories:
 
 | **Function** | **Description** | **Available formats** | **Migration model** | **Available documentation** |
 | --- | --- | --- | --- | --- |
-| --- | --- | --- | **Inbound** :  <br> **Oubound** :   | []() |
-| --- | --- | --- | **Inbound** :  <br> **Oubound** :   | []() |
-
-
+| Standard VM images | Import and export VM images in standard formats supported by the OVF hypervisor | .OVF | **Inbound** : import images via API or User Interface <br>**Outbound** : export VM images which can be reused on any compatible environment | [The fundamentals of VCD](/pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vcd-get-concepts) <br><br>[VMware Cloud Director - Migrate from VMware vSphere on OVHcloud ](/pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vcd_migration_use-cases) <br><br>[OVF Tool](/pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/ovf_tool) |
 
 ## 2-OVHcloud Implementations
 
 | **Function** | **Description** | **Available formats** | **Migration model** | **Available documentation** |
 | --- | --- | --- | --- | --- |
-| --- | --- | --- | **Inbound** :  <br> **Oubound** :   | []() |
-| --- | --- | --- | **Inbound** :  <br> **Oubound** :   | []() |
+| Management with OVHcloud API | VM management via API | JSON, YAML, scripts | **Inbound**: automated deployment, management and import <br>**Outbound**: export configurations, scripts and automations that can be used on other platforms | [Login to OVH API](/pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/login_a_l_api_ovh) |
+| Manual backup and restore | Snapshots, backups and restores on demand via User Interface or API | Snapshots, disk images | **Inbound**: standard VM Image Import <br>**Outbound**: no access to native backup files. You cannot export to a new hosting environment | [Backups with Veeam and restorations](/pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vcd-backup) |
 
 
 ## 3-Specific features
 
 | **Function** | **Description** | **Available formats** | **Migration model** | **Available documentation** |
 | --- | --- | --- | --- | --- |
+| Virtual network management | Network configuration (subnets, security groups) specific to OVHcloud | NA | **Inbound**: adapt network configurations to the OVHcloud environment. <br>**Outbound**: | [VMware Cloud Director - Network Concepts and Best Practices]() <br><br>[VMware Cloud Director - Creating Network Components via Public VCF as-a-Service]() |
```

Comment:
Hey @H4516022Y are there supposed to be links here? I did not find a guide with this title "VMware Cloud Director - Creating Network Components via Public VCF as-a-Service" only found this one "VMware Cloud Director - Creating network components from the Public VCF as-a-Service control panel".